Sonarqube-code-review-best-practices.html

```
@@ -0,0 +1,2351 @@
+
+        .hero h1 {
+          font-size: 1.7rem;
+        }
+        .toc-grid {
+          grid-template-columns: 1fr;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <button class="sidebar-toggle" id="sidebarToggle" aria-label="メニューを開閉">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)